eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -85,6 +85,11 @@ jobs:
         # Limiting interp runs as we don't need as much coverage.
         - Debian.9.Amd64.Open
 
+    # OSX arm64
+    - ${{ if eq(parameters.platform, 'OSX_arm64') }}:
+      - ${{ if eq(parameters.jobParameters.isFullMatrix, true) }}:
```

Comment:
I know we discussed this earlier, but I believe we should remove this condition for libraries and then instead condition the test runs in the yml entry point?